bigquery_etl/cli/utils.py

```diff
@@ -44,7 +45,11 @@ def is_authenticated():
     try:
         bigquery.Client(project="")
     except DefaultCredentialsError:
-        return False
+        try:
+            subprocess.run(["gcloud", "auth", "application-default", "login"])
+        except Exception as e:
+            print(f"Could not log in to GCP: {e}")
+            return False
```

Comment:
this will prompt the browser authentication flow

Reply:
Also, I believe `check=True` needs to be specified for `subprocess.run()` to raise an exception if the command exited with a non-zero exit code.

Reply:
I opened https://github.com/mozilla/bigquery-etl/issues/5174 for now, to keep track of this. I think for now we just keep the flow as is, and ask users to run `gcloud auth` themselves.